### DISCUSSION TITLE
How to change the "Files extensions list" for Single Page App

### DISCUSSION THREAD

**Discussion by kwokhou** [getlift/lift#240] [Constructs]
Is it possible to change the CloudFront function's "Files extensions list" that what automatically created?

https://github.com/getlift/lift/blob/c0f260ad59bb1d4c08a15083caf0803e3baf027d/src/constructs/aws/SinglePageApp.ts#L47

I'd like to add "docx" to the list

**Comment by mnapoli** [comment]
Hi, a pull request is welcome!
